parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -225,7 +247,9 @@ where
 	}
 }
 
-impl<CreateAssetCall: Get<CallIndex>> MessageToXcm<CreateAssetCall> {
+impl<CreateAssetCall: Get<CallIndex>, FeeGetter: XcmFeeGetter>
+	MessageToXcm<CreateAssetCall, FeeGetter>
+{
```

Comment:
I think using `where` is a little bit easier to read.
```suggestion
impl<CreateAssetCall, FeeGetter> MessageToXcm<CreateAssetCall, FeeGetter>
where  
    CreateAssetCall: Get<CallIndex>,
    FeeGetter: XcmFeeGetter,
{
```